python/tvm/autotvm/task/topi_integration.py

```diff
@@ -281,6 +284,21 @@ def _topi_nn_conv2d_NCHWc(*args, **kwargs):
             s = topi.generic.schedule_conv2d_NCHWc([C])
             return s, [A, W, C]
 
+        @register("topi_nn_softmax")
+        def _topi_nn_softmax(*args, **kwargs):
+            assert not kwargs, "Do not support kwargs in template function call"
+            args = deserialize_args(args)
+            if len(args) > 1:
+                x, axis = args[:2]
+            else:
+                x = args[0]
+                axis = None
+            C = topi.nn.softmax(*args, **kwargs)
+            s = topi.generic.schedule_softmax([C])
+            if axis is not None:
+                return s, [x, axis, C]
```

Comment:
`axis` is not a tensor, so we don't need to return it here

Reply:
Updated